replica/compaction_group.hh

```diff
@@ -190,7 +190,9 @@ public:
     compaction_group_ptr& main_compaction_group() noexcept;
     compaction_group_ptr& select_compaction_group(locator::tablet_range_side) noexcept;
 
-    utils::small_vector<compaction_group*, 3> compaction_groups() noexcept;
+    uint64_t live_disk_space_used() const noexcept;
+
+    utils::small_vector<compaction_group*, 3> compaction_groups() const noexcept;
```

Comment:
It's kind-of-wrong to return non-const compaction groups from a const accessor.

Reply:
Agreed.

Reply:
and...?